assets/wire/components/fields/CharCount.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {characterCount, gettext} from 'utils'
```

Comment:
would be good to keep it consistent and have `;` here, probably with eslint rule to check it

Reply:
the rule already exists on our config, I'll run `eslint --fix` once the PR is reviewed 👍🏻